internal/generate/olm-catalog/package_manifest.go

```diff
@@ -26,16 +26,18 @@ import (
 	"github.com/operator-framework/api/pkg/validation"
 	"github.com/operator-framework/operator-registry/pkg/registry"
 	"github.com/operator-framework/operator-sdk/internal/generate/gen"
+	"github.com/operator-framework/operator-sdk/internal/scaffold"
 	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
 
 	"github.com/ghodss/yaml"
 	log "github.com/sirupsen/logrus"
 )
 
 const (
-	PackageManifestFileExt = ".package.yaml"
+	packageManifestFileExt = ".package.yaml"
 
-	ManifestsDirKey = "manifests"
+	// TODO: Is this an input that needs to be set via the csv generator inputs flag?
```

Comment:
@estroz

Reply:
Looks like it's the same thing here as with `BundleDirKey `.
Not really a generator input that ever needs to be set. 
Only place this is actually used is in `package_manifest_test.go` to have the generator point to  an existing package manifest data directory.
```Go
inputDir := filepath.Join(testGoDataDir, OLMCatalogDir, testProjectName)
cfg := gen.Config{
	OperatorName: testProjectName,
	Inputs:       map[string]string{manifestsDirKey: inputDir},
}
``` 
Going to replace this with a private field in `pkgGenerator`.